com.unity.netcode.gameobjects/Tests/Runtime/Helpers/NetworkManagerHelper.cs

```diff
@@ -68,6 +70,7 @@ public static bool StartNetworkManager(out NetworkManager networkManager, Networ
 
                 Debug.Log($"{nameof(NetworkManager)} Instantiated.");
 
+#if UNITY_UNET_PRESENT
```

Comment:
This part of the back port has to be different because of the fixes applied to develop for the console testing (#1664 & #1653).  I will be migrating these updates into develop next week.  Until then, this prevents certain tests from failing when UNet is not included and uses only SIPTransport.